src/tests/ftest/daos_test/daos_core_test-rebuild.yaml

```diff
@@ -69,10 +69,11 @@ daos_tests:
       daos_test: r
       test_name: rebuild tests 17
       args: -s3 -u subtests="17"
-    test_r_18:
-      daos_test: r
-      test_name: rebuild tests 18
-      args: -s3 -u subtests="18"
+    # Moved to weekly. DAOS-5717
+    # test_r_18:
+    #   daos_test: r
+    #   test_name: rebuild tests 18
+    #   args: -s3 -u subtests="18"
```

Comment:
Minor: If you have to rebase, can we completely remove these lines instead of leaving them commented out? If there is some specific reason, then no worries.

Reply:
This move is temporary. I think the bring-back process will be easier if we just have to uncomment.

Reply:
`git revert` is usually what one does to undo something like this.

Reply:
I see. Thanks for suggestion.